README.md

```diff
@@ -4,16 +4,11 @@ A gateway which uses graphql mesh to split a query into smaller queries to the d
 
 ### Relevant for deployment
 
-| Name             | Description             | Value in Dev Environment | Value in Prod Environment |
-|------------------|-------------------------|--------------------------|---------------------------|
-| GATEWAY_PORT     | PostgreSQL database URL | 0.0.0.0                  | 0.0.0.0                   |
-| GATEWAY_HOSTNAME | Database username       | 8080                     | 8080                      |
-| DAPR_HTTP_PORT   | Dapr HTTP Port          | -                        | 3500                      |
-
-
-### Other properties
 | Name                   | Description                        | Value in Dev Environment                                                   | Value in Prod Environment                                           |
 |------------------------|------------------------------------|----------------------------------------------------------------------------|---------------------------------------------------------------------|
+| GATEWAY_PORT           | PostgreSQL database URL            | 0.0.0.0                                                                    | 0.0.0.0                                                             |
```

Comment:
Sorry I just saw. These description are completely wrong for the first two. Also they are mixed up. The port has a host as value and the host has a port as value??